### ISSUE TITLE
[WinError 32] The process cannot access the file because it is being used by another process

### ISSUE THREAD

**Issue by Taikono-Himazin** [django-compressor/django-compressor#1097]
environment:
django=3.2.11
django-compressor=3.1

I'm trying to use it with the following settings:
```
STATIC_ROOT = os.path.join(BASE_DIR, 'static')
STATIC_URL = '/' + URL_PREFIX + 'static/'

# django-compressor
COMPRESS_ENABLED = not DEBUG
COMPRESS_OFFLINE = True
if COMPRESS_ENABLED:
    STATICFILES_FINDERS = [
        'django.contrib.staticfiles.finders.FileSystemFinder',
        'django.contrib.staticfiles.finders.AppDirectoriesFinder',
        'compressor.finders.CompressorFinder',
    ]
    COMPRESS_FILTERS = {
        'css': ['compressor.filters.cssmin.rCSSMinFilter'],
        'js': ['compressor.filters.jsmin.JSMinFilter']
    }
```

However, I get the following error and cannot "offline compression".
command:
`python manage.py compress --traceback`

console
```
[DEBUG] 2022-01-13 14:51:48,199 'Using selector: SelectSelector' asyncio
Compressing... Traceback (most recent call last):
Traceback (most recent call last):
  File "[prject_root]\manage.py", line 21, in <module>
    main()
  File "[prject_root]\manage.py", line 17, in main
    execute_from_command_line(sys.argv)
  File "[python_env]\lib\site-packages\django\core\management\__init__.py", line 419, in execute_from_command_line
    utility.execute()
  File "[python_env]\lib\site-packages\django\core\management\__init__.py", line 413, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "[python_env]\lib\site-packages\django\core\management\base.py", line 354, in run_from_argv
    self.execute(*args, **cmd_options)
  File "[python_env]\lib\site-packages\django\core\management\base.py", line 398, in execute
    output = self.handle(*args, **options)
  File "[python_env]\lib\site-packages\compressor\management\commands\compress.py", line 296, in handle
    self.handle_inner(**options)
  File "[python_env]\lib\site-packages\compressor\management\commands\compress.py", line 319, in handle_inner
    offline_manifest, block_count, results = self.compress(engine, extensions, verbosity, follow_links, log)
  File "[python_env]\lib\site-packages\compressor\management\commands\compress.py", line 227, in compress
    raise errors[0]
django.core.management.base.CommandError: An error occurred during rendering [template_file]: 
[WinError 32] The process cannot access the file because it is being used by another process: '[prject_root]\\static\\CACHE\\css\\output.f0d4b7a00b85.css'
```

The template name is different each time.
What should I do.
If there is any other necessary information, I will give it out.

**Comment by diox** [comment]
Hi,

This should be fixed by https://github.com/django-compressor/django-compressor/commit/00910e0bd5295044befacdad7b88249ec7bbf142, but I don't have a Windows environment to test it. Could you test with the `develop` branch and report back ? Thanks.

**Comment by Taikono-Himazin** [comment]
thank you!  
I haven't tried it yet, but I'd love to try it next week.  
I will contact you again.

**Comment by diox** [comment]
This should be fixed in 4.0.
